docs/manifest/schema.rst

```diff
@@ -81,3 +83,22 @@ Manifest Schema
             arguments:   # arguments that override the global script arguments
                          # (optional)
                 <key>: <value>
+
+            max_runtime: HH:MM,   # max runtime in hours:minutes (optional)
+
+            interest_list:       # list of email addresses (optional)
+                - list@domain
+                - user@domain
+
+            priority: #  job priority (integer) (optional)
```

Comment:
lower = top or highest = top ?

And how is it used?

Reply:
added prio description